packages/vtree/core/modeling/operations/booleans/intersect.js

```diff
@@ -0,0 +1,8 @@
+const { toArray } = require('@jscad/array-utils')
+
+const intersection = (...solids) => {
+  solids = toArray(solids)
```

Comment:
Shouldn’t these signatures match with modeling, and use flatten()?
The use of ...solids already guarantees that solids will be an array.

Reply:
agreed